### PR TITLE
Fix

### PR DESCRIPTION
**Pull request by MaksGMaks**

### CODE REVIEW

File: _client/src/Ui/Dialogs/DialogManager.cpp [hunk 1]
```diff
@@ -14,6 +14,7 @@ DialogManager::~DialogManager()
     delete m_loginDialog;
     delete m_registrationDialog;
     delete m_messageDialog;
+    m_messageDialog->addButton(QMessageBox::Ok);
```
Comment (Sage-Cat) [inline _client/src/Ui/Dialogs/DialogManager.cpp]: WTF? Why in destructor? Did you test this?

**Review by Sage-Cat — CHANGES_REQUESTED** [review]

File: _client/src/Ui/Dialogs/DialogManager.cpp (hunk 1 above)
Comment (MaksGMaks) [reply _client/src/Ui/Dialogs/DialogManager.cpp]: didn't see it. It had to be in initDialogs

**Review by Sage-Cat — APPROVED** [review]
LGTM

**Review by DanyloPlotskykh — APPROVED** [review]

**Review by Field-Effect-Transistor — CHANGES_REQUESTED** [review]
looks good

**Review by Field-Effect-Transistor — APPROVED** [review]

**Pull request merged**